src/rabbit_watchdog_disk.erl

```diff
@@ -92,7 +92,7 @@ check_forbidden(Dir, FDirs) ->
     end.
 
 check(Dir, FDir) ->
-    case re:run(Dir, FDir, [{capture, none}]) of
+    case re:run(Dir, FDir ++ "/?$", [{capture, none}]) of
```

Comment:
I'd say subdirs can be deleted. Originally they were forbidden too.

Reply:
Ok. Looks good. My concern is about other important system DIR's. Can we forbid directory + sub-directories as well for "some" directories like `/bin`, `proc`, `/sbin`, ... which the OS heavily relies on for its operation. If users need to delete such directories + their sub-directories in there as well, then they would have to do it manually, without blaming the plugin :) how does that sound?

Reply:
Or, we can have a `soft` and `hard` severity option, to decide if ability to delete will allow sub-directories or not (default to `soft` if user didnt specify). e.g. 
`{"Disk Watchdog", rabbit_watchdog_disk, 5000, [{dirs, [ "/tmp", "/another-directroy-to-clear" ], {severity, hard}}]}`

Reply:
That's perfect. :) We have to make it hard for them to crash their systems.
I'll update the regexes.

Reply:
Ok, having it more fine grained per DIR, and not global would sure grant even more control/flexibility to the user (in case user wants to define config consistently, if tuple without `force`, e.g. `{"/home/rabbit", soft_or_any_word_or_typo}` is specified, it shouldn't fail but treat it as not-hard/not-forced). Maybe supporting `{DIR, force | hard}` could be worthwhile? But definitely sounds good! :) 👍

Reply:
Totally agree. I pushed the new changes according to this discussion. ;) It now accepts force | hard and handles all other modifiers safely.